tests/chains/ethereum/scripts/cmd/helper/chain.go

```diff
@@ -0,0 +1,217 @@
+package helper
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"math/big"
+	"os"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+	gethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/client"
+	ibccommitment "github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ibccommitmenttesthelper"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ibchandler"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ics20bank"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ics20transferbank"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/simpletoken"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/relay/ethereum"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/wallet"
+	"github.com/hyperledger-labs/yui-relayer/core"
+)
+
+type PathConfig struct {
+	Src core.PathEnd `json:"src"`
+	Dst core.PathEnd `json:"dst"`
+}
```

Comment:
You can replace `PathConfig` with `core.Path` from `yui-relayer/core/path.go`.

Reply:
Replace PathConfig to core.Path
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/e05d413e4fefcda5b89b39f5b927b27876ba119f